NBitcoin/Protocol/NodeServer.cs

```diff
@@ -266,27 +267,22 @@ internal void ExternalAddressDetected(IPAddress iPAddress)
 		{
 			if(!ExternalEndpoint.Address.IsRoutable(AllowLocalPeers) && iPAddress.IsRoutable(AllowLocalPeers))
 			{
-				NodeServerTrace.Information("New externalAddress detected " + iPAddress);
+				Logs.NodeServer.LogInformation("New externalAddress detected {externalAddress}", iPAddress);

```

Comment:
You forgot the $ sign in front of the string

Reply:
this is still an issue ^

Reply:
no, actually he wants to use structured logging